app/main.py

```diff
@@ -2,24 +2,74 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+        self.value = None
+
+    def __get__(self, instance: object, owner: type) -> int:
+        return self.value
+
+    def __set__(self, instance: object, value: int) -> None:
+        if not isinstance(value, int):
+            raise TypeError("Value must be an integer.")
+        if value < self.min_amount or value > self.max_amount:
+            raise ValueError(f"Value must be between "
+                             f"{self.min_amount} and {self.max_amount}.")
+        self.value = value
+
+    def __set_name__(self, owner: type, name: str) -> None:
+        self.name = name
 
 
 class Visitor:
-   pass
+    def __init__(self, name: str, age: int, weight: int, height: int) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
```

Comment:
The attributes `age`, `weight`, and `height` in the `Visitor` class are being set directly, bypassing the `IntegerRange` descriptor. Consider using the descriptor to manage these attributes to ensure the validation logic is applied.